db-service/lib/cql-functions.js

```diff
@@ -51,7 +51,7 @@ const StandardFunctions = {
    * @param {string} x
    * @returns {string}
    */
-  count: x => `count(${x || '*'})`,
+  count: x => `count${(x?.xpr && x ) || (x && `(${x})`) || '(*)'}`,
```

Comment:
```suggestion
  count: x => !x ? 'count(*)' : x.xpr ? `count${x}` : `count(${x})`,
```

Reply:
What is an example of such expressions in `count()`?

Reply:
Looks like this is a follow up fix for always wrapping `xpr` in `expr()`? → likely not good